src/locale/messages.fr.xlf

```diff
@@ -728,100 +723,161 @@
         <note priority="1" from="description">faq answer</note>
       </trans-unit>
-      <trans-unit id="57c82df8ab9888592cfd3419caf75ec72c794db6" datatype="html">
-        <source>When a domain name (such as &apos;zonemaster.net&apos;) is submitted to Zonemaster (using CLI or GUI), it will verify the domain name’s general health with a series of tests. The tests conducted by Zonemaster can be found in <x id="START_LINK" ctype="x-a" equiv-text="&lt;a href=&quot;https://github.com/zonemaster/zonemaster/tree/master/docs/public/specifications/tests#list-of-defined-test-cases&quot;&gt;"/>the Defined Test Cases<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a&gt;"/> document.</source>
+      <trans-unit id="333892b5f3094578ceccffb29e7498c1c5fdf3b5" datatype="html">
+        <source><x id="START_LINK" ctype="x-a" equiv-text="&lt;a href=&quot;https://github.com/zonemaster/zonemaster-engine&quot;&gt;"/>GUI<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a&gt;"/>, a web interface to the Backend.</source>
+        <target state="translated"><x id="START_LINK" ctype="x-a" equiv-text="&lt;a href=&quot;https://github.com/zonemaster/zonemaster-engine&quot;&gt;"/>GUI<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a&gt;"/>, une interface web au Backend.</target>
         <note priority="1" from="description">faq answer</note>
       </trans-unit>
-      <trans-unit id="1072f67587614ba02bf775b31fcf47604c0a74d9" datatype="html">
-        <source>Users who are knowledgable about the DNS protocol.</source>
+      <trans-unit id="7aec1bc450ef7a1d3fbec65dc277ea4564f25c9e" datatype="html">
+        <source>The components will help different types of users to check domain servers for configuration errors and generate a report that will assist in fixing the errors.</source>
+        <target state="translated">Ces composants aideront différents types d&apos;utilisateurs et utilisatrices à vérifier les erreurs de configuration des serveurs de domaine et à générer un rapport qui aidera à corriger les erreurs.</target>
         <note priority="1" from="description">faq answer</note>
       </trans-unit>
-      <trans-unit id="bcc9e8cca9979fe26d8cc69c1b7dda20d67fb6ec" datatype="html">
-        <source>Secondly, all tests that Zonemaster runs are defined in Test Case specifications that can be found in the <x id="START_LINK" ctype="x-a" equiv-text="&lt;a href=&quot;https://github.com/zonemaster/zonemaster/tree/master/docs/public/specifications/tests#list-of-defined-test-cases&quot;&gt;"/>Defined Test Cases<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a&gt;"/> document.</source>
+      <trans-unit id="fdc6b1d33f90c73b6ed81e0b6af07e4109d4dea7" datatype="html">
+        <source>DNS administrators, experts and beginners alike, who want to check their DNS configuration;</source>
+        <target state="translated">Administrateurs et administratrices DNS, expertes comme débutants, qui veulent vérifier leur configuration DNS ;</target>
```

Comment:
From what I can guess, this string is displayed in a bullet point list right next to “users who want to know whether the domain name they own or use have any issues or not”, isn’t it?
Shouldn’t it be translated as:
```suggestion
        <target state="translated">les administrateurs et administratrices DNS, expertes comme débutants, qui veulent vérifier leur configuration DNS ;</target>
```

Reply:
Good catch, I got distracted by the capitalized DNS ^^'